utils/exchange-bootstrap.sh

```diff
@@ -119,42 +135,53 @@ then
 		"https://api.github.com/repos/${EXCHANGE_ORG}/${REPO_NAME}/hooks"
 fi
 
-# CircleCI: follow the project
-echo "CircleCI: Following the project"
-curl -sS --fail -X POST "https://circleci.com/api/v1.1/project/github/${EXCHANGE_ORG}/${REPO_NAME}/follow?circle-token=${CIRCLECI_TOKEN}"
+# NO longer needed, this API request fails everytime we call it
+# # CircleCI: follow the project
+# echo "CircleCI: Following the project"
+# curl -v -sS --fail -X POST \
+#      --header "Circle-Token: ${CIRCLECI_TOKEN}" \
+#      "https://circleci.com/api/v1.1/project/github/${EXCHANGE_ORG}/${REPO_NAME}/follow"
+
 
 # CircleCI: upload the read-write key
 echo "CircleCI: Adding read-write SSH key"
 curl -sS --fail -X POST --header "Content-Type: application/json" \
+  --header "Circle-Token: ${CIRCLECI_TOKEN}" \
 	-d '{"hostname":"github.com","private_key":"'"$(cat "/tmp/${PACK}_rsa")"'"}' \
-	"https://circleci.com/api/v1.1/project/github/${EXCHANGE_ORG}/${REPO_NAME}/ssh-key?circle-token=${CIRCLECI_TOKEN}"
+	"https://circleci.com/api/v1.1/project/github/${EXCHANGE_ORG}/${REPO_NAME}/ssh-key"
 
 # CircleCI: specify the credentials (the machine login and the new user-scope token)
```

Comment:
Setting MACHINE_USER and MACHINE_PASSWORD is handled well by `tools/reset_github_user_token_and_update_circleci.sh`. We should standardize on one script so we don't duplicate these calls in both.